src/engine/renderer/glsl_source/dispersion_C_fp.glsl

```diff
@@ -37,26 +37,26 @@ DECLARE_OUTPUT(vec4)
 void	main()
 {
 	// compute incident ray
-	vec3 I = normalize(var_Position - u_ViewOrigin);
+	vec3 incicentRay = normalize(var_Position - u_ViewOrigin);
```

Comment:
spelling

Reply:
good catch!

Reply:
I don't know what can we do to improve glsl debugging, stuff get easily unnoticed…